packages/vue/src/index.ts

```diff
@@ -128,8 +136,8 @@ export type {
 	StackedAreaChartOptions,
 	TreeChartOptions,
 	TreemapChartOptions,
-	WorldCloudChartOptions
```

Comment:
we should export both and mark world as deprecated

Reply:
Updated